suite-native/atoms/src/TouchableSwitchRow.tsx

```diff
@@ -0,0 +1,70 @@
+import { ReactNode } from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { Icon, IconName } from '@suite-native/icons';
+
+import { Box } from './Box';
+import { Card } from './Card/Card';
+import { HStack, VStack } from './Stack';
+import { Switch } from './Switch';
+import { Text } from './Text';
+
+type TouchableSwitchRowProps = {
+    isChecked: boolean;
+    onChange: (value: boolean) => void;
+    text: ReactNode;
+    accessibilityLabel: string;
+    description?: ReactNode;
+    icon: IconName;
+    testID?: string;
```

Comment:
Based on the layout (and similar `CardWithIconLayout`), this order might seem more natural:
```suggestion
    icon: IconName;
    accessibilityLabel: string;
    text: ReactNode;
    description?: ReactNode;
    isChecked: boolean;
    onChange: (value: boolean) => void;
    testID?: string;
```

Reply:
And I thought that I am nitpicky 🤣 Fixed.